main.js

```diff
@@ -67,11 +70,24 @@ async function main() {
       $('#face-touch-alert').hide();
     }
   }
+
+  function getHowl(name) {
+    if (name === 'pop') {
+      return new Howl({ src: [popUrl], html5: true });
+    } else if (name === 'coronavirus') {
+      return new Howl({ src: [coronavirusUrl], html5: true });
+    } else {
+      return null;
+    }
+  }
+
+  const heatmapCookie = Cookies.get('heatmap') === 'true'
```

Comment:
Do we need a helper function to get cookies and set it immediately to keep them from expiring? This is the use case for when a user didn't change the settings for 30 days would but keep visiting the site? Would his setting be reset?

Reply:
Yeah that's important. From what I found [here](https://thoughtbot.com/blog/lucky-cookies) cookies will be removed from the browser when they expire.  So we should either make the expiration date the maximum (I think it's 10,000 days) or better would be a helper method to reset the cookies expiration on each run. From what I found, we can't get the expiration date of the cookies, so we'd need to simply reinitialize them every time the user visits. I created a function `initializeCookie(name)` to do just this.

Reply:
Either way seems fine. Now that you mentioned the maximum value, it seems like a simpler method?

Reply:
Yup! I think what we currently have works well. I like that if users don't visit for the number of days their settings aren't saved, but that counter will reset if they visit within that period.

Reply:
One other issue before merging is that audio notifications don't seem to work if I'm in a Zoom call? Either that or I broke it when setting these cookies 😄. I'll test when the call finishes.

Reply:
I think that might be due to the audio pool internal in Howl js. But I haven't got time to look at it.